ui/admin/app/templates/scopes/scope/aliases/alias/index.hbs

```diff
@@ -23,12 +23,76 @@
       </nav.link>
     </Rose::Nav::Tabs>
   </page.navigation>
-
+  {{#if (can 'delete model' @model)}}
+    <page.actions>
+      <Hds::Dropdown as |dd|>
+        <dd.ToggleButton
+          data-test-manage-dropdown
+          @text={{t 'actions.manage'}}
+          @color='secondary'
+        />
+        <dd.Interactive
+          data-test-manage-alias-dropdown-delete
+          @color='critical'
+          @text={{t 'actions.delete'}}
+          {{on 'click' (fn this.aliases.toggleModal @model)}}
```

Comment:
Hmm the one from managed targets has two dropdown options for both clear and delete, how come these ones are different and both options are in a modal when you "delete"?